whatlies/transformers/__init__.py

```diff
@@ -1,12 +1,29 @@
 from whatlies.transformers._transformer import Transformer
-from whatlies.transformers._pca import Pca
+
+# from whatlies.transformers._pca import Pca
 from whatlies.transformers._umap import Umap
 from whatlies.transformers._noise import Noise
 from whatlies.transformers._addrandom import AddRandom
 from whatlies.transformers._tsne import Tsne
 from whatlies.transformers._normalizer import Normalizer
 from whatlies.error import NotInstalled
 
+
+from whatlies.transformers._transformer import SklearnTransformer
+from sklearn.decomposition import PCA
+
+
+def Pca(n_components=2, *args, **kwargs):
+    """
+    Creates a PCA transformer component that you can use on an `EmbeddingSet`.
+
+    You can pass it all the parameters from the underlying scikit-learn implementation for customisation.
+    """
+    return SklearnTransformer(
+        PCA, name="pca", n_components=n_components, *args, **kwargs
```

Comment:
The `name` should be passed as positional, not keyword argument here. Otherwise, the `*args` could not be used at all and becomes useless. For example, with current implementation, the following would raise an error:
```python
Pca(2, 0, 1, key='dummy')
```

Reply:
This is making me wonder. Do we *really* want to allow for args here? There's something to be said to force the users to only pass arguments explicitly.

Reply:
I am also in favor of having only the `**kwargs`, since the positional arguments are usually important and therefore should be explicitly listed in the signature. In my opinion, the (only and) most useful case for having both is when the function is defined as `def func(*args, **kwargs): ...` with no additional arguments.

Reply:
Agreed! I also think on the scikit-learn side of things they are becomming more strict when it comes to arguments. I recall version 0.24.0 onwards the API demands key-word arguments for gridsearch.